### PR TITLE
Verification

### PR DESCRIPTION
**Pull request by kknevitt**
- Adding to `authentication` to provide details on how to verify signatures with webhook payloads
- Replacing references to api keys with api tokens for a consistent description throughout.

### CODE REVIEW

File: source/includes/_authentication.md [hunk 1]
```diff
@@ -9,41 +9,141 @@ curl "https://api.handshq.com/v1/authentication" \
...
+    # compare values
+    Rack::Utils.secure_compare(generated_signature, provided_signature)
+  end
+```
+> Different frameworks will have their own ways of achieving the workflow as described, and will have their own considerations of best practices, for example in this case the use of `#secure_compare` for additional protection against timing attacks.
+
+Once you have an endpoint which is able to receive notifications and have registered an `event subscription` with the URL pointing to this endpoint, you can do the following to verify the signature.
+
+- Obtain the value of the signature from the `X-Handshq-Webhook-Signature` header.
+- Generate a HMAC hex digest with the details as described [here](#signature-details)
```
Comment (hughbric) [inline source/includes/_authentication.md:140]: Cool haven't seen this md before, does it just link to the heading of `signature-details`, does it matter that Signature details is an `h3`?

Comment (kknevitt) [reply source/includes/_authentication.md:140]: Shouldn't do, the id attribute is being automatically added (presumably by slate or some underlying md processor) based off the content of the heading 👍

File: source/includes/_authentication.md [hunk 2]
```diff
@@ -9,41 +9,141 @@ curl "https://api.handshq.com/v1/authentication" \
...
+### Verification strategy - Signed Requests
+<!-- Credit to @jasny for putting together a gist with different example implementations in different languages -->
+
+> All examples assume a key of "my_key" and JSON data of {"bar":"foo"}
+
+> Ruby Example:
+
+```ruby
+  require 'openssl'
+  key = "my_key"
```
Comment (jodydeanrogan) [inline source/includes/_authentication.md:62]: does it need to be clear that the key would actually be the HandsHQ API token here?

**Review by jodydeanrogan — APPROVED** [review]
very nice, very clear (not easy to write language agnostic documentation), just one minor comment, all good 👍

**Pull request merged**